source/deploying-with-ansible/guide/install-opendistro-server.rst

```diff
@@ -0,0 +1,565 @@
+
+	.. code-block:: none
+		:class: output
+
+		192.168.0.108 | SUCCESS => {
+		    "changed": false,
+		    "ping": "pong"
+		}
+		192.168.0.180 | SUCCESS => {
+		    "changed": false,
+		    "ping": "pong"
+		}
+
+
+1 - Access to wazuh-ansible
+---------------------------
+
+1.1 - We access the directory where we have cloned the repository from our Ansible server.
+
+.. code-block:: console
+
+	ansible@ansible:/etc/ansible/roles/wazuh-ansible$ ls
+
+.. code-block:: none
+	:class: output
+
+	CHANGELOG.md  playbooks  README.md  roles  VERSION
+
+We can see the roles we have.
+
+.. code-block:: console
+
+	ansible@ansible:/etc/ansible/roles/wazuh-ansible$ tree roles -d
+
+.. code-block:: none
+	:class: output
+
+	roles
+	├── ansible-galaxy
+	│   └── meta
+	├── elastic-stack
+	│   ├── ansible-elasticsearch
+	│   │   ├── defaults
+	│   │   ├── handlers
+	│   │   ├── meta
+	│   │   ├── tasks
+	│   │   └── templates
+	│   └── ansible-kibana
+	│       ├── defaults
+	│       ├── handlers
+	│       ├── meta
+	│       ├── tasks
+	│       └── templates
+	├── opendistro
+	│   ├── opendistro-elasticsearch
+	│   │   ├── defaults
+	│   │   ├── handlers
+	│   │   ├── meta
+	│   │   ├── tasks
+	│   │   └── templates
+	│   └── opendistro-kibana
+	│       ├── defaults
+	│       ├── handlers
+	│       ├── tasks
+	│       ├── templates
+	│       └── vars
+	└── wazuh
+		├── ansible-filebeat
+		│   ├── defaults
+		│   ├── handlers
+		│   ├── meta
+		│   ├── tasks
+		│   └── templates
+		├── ansible-filebeat-oss
+		│   ├── defaults
+		│   ├── handlers
+		│   ├── meta
+		│   ├── tasks
+		│   └── templates
+		├── ansible-wazuh-agent
+		│   ├── defaults
+		│   ├── handlers
+		│   ├── meta
+		│   ├── tasks
+		│   └── templates
+		└── ansible-wazuh-manager
+			├── defaults
+			├── files
+			├── handlers
+			├── meta
+			├── tasks
+			├── templates
+			└── vars
+
+
+And we can see the preconfigured playbooks we have.
+
+.. code-block:: console
+
+	ansible@ansible:/etc/ansible/roles/wazuh-ansible$ tree playbooks/
+
+.. code-block:: none
+	:class: output
+
+	playbooks/
+	├── wazuh-agent.yml
+	├── wazuh-elastic_stack-distributed.yml
+	├── wazuh-elastic_stack-single.yml
+	├── wazuh-elastic.yml
+	├── wazuh-kibana.yml
+	├── wazuh-manager-oss.yml
+	├── wazuh-manager.yml
+	├── wazuh-odfe-production-ready.yml
+	├── wazuh-odfe-single.yml
+	├── wazuh-opendistro-kibana.yml
+	└── wazuh-opendistro.yml
+
+
+Using **Opendistro** and **Kibana** roles, we will install and configure the Open Distro for Elasticsearch server components. There are several variables we can use to customize the installation or configuration. To consult the default configuration go to this :ref:`section <wazuh_ansible_reference>`.
+
+If we want to change the default configuration, we can change the following files:
+- ``/etc/ansible/roles/wazuh-ansible/roles/opendistro/opendistro-elasticsearch/defaults/main.yml``
+- ``/etc/ansible/roles/wazuh-ansible/roles/opendistro/opendistro-kibana/defaults/main.yml``
```

Comment:
Fix this, this must be show in different lines

![image](https://user-images.githubusercontent.com/14913942/141313809-ab3b6a53-2e79-4cdb-8816-38001911c7af.png)

Reply:
Done
![image](https://user-images.githubusercontent.com/8108052/141551981-4682a4e3-49a3-4a85-97d5-bcd26dcf1d39.png)